docs/learn/expert-guides/deploy-up-with-lsp23.md

```diff
@@ -433,17 +454,10 @@ async function main() {
   );
   await tx.wait(1);
 }
-
-// We recommend this pattern to be able to use async/await everywhere
-// and properly handle errors.
-main().catch((error) => {
-  console.error(error);
-  process.exitCode = 1;
-});
 ```
 
 </details>
 
-### Run the script
+### Conclusion
 
-You can now run the script using `npx hardhat run scripts/deploy.ts --network <networkName>`.
+This is just an example on how you could implement the deployment of a Universal Profile and its Key Manager using the LSP23 Linked Contracts Factory. You can use this script as a base to create your own deployment script that would fit more your need. You can also refer to the [Lukso Playground](https://github.com/lukso-network/lukso-playground) repository to see how it could be implemented using Hardhat.
```

Comment:
```suggestion
This is just an example on how you could implement the deployment of a Universal Profile and its Key Manager using the LSP23 Linked Contracts Factory. You can use this script as a base to create your own deployment script that would fit more your need. You can also refer to the [LUKSO Playground](https://github.com/lukso-network/lukso-playground) repository to see how it could be implemented using Hardhat.
```